packages/hub/routes/job-tickets.ts

```diff
@@ -17,6 +17,19 @@ export default class JobTicketsRoute {
     autoBind(this);
   }
 
+  async list(ctx: Koa.Context) {
+    if (!ensureLoggedIn(ctx)) {
+      return;
+    }
+
+    let prisma = await this.prismaManager.getClient();
+    let jobTickets = await prisma.jobTicket.findMany({ where: { ownerAddress: ctx.state.userAddress } });
```

Comment:
Do we need any guarantees of order here? And what happens if a user has hundreds of job tickets?

Reply:
This is definitely the most naïve possible implementation, we can refine as needed 😀